action.yaml

```diff
@@ -278,34 +365,54 @@ runs:
         echo "files changed: $files"
         dags_only=1
 
+        skip_deploy=1
+
+        # This for loop checks for following cases:
+        # 1. If no file is part of the input root folder, then it skips deploy
```

Comment:
In order to accommodate for the case when dbt project and astro project are part of the same repo, we would need to ensure that we only trigger image deploy when any file is updated inside the astro project's root folder, this is slightly different from existing behaviour where if any file outside the `dags` folder is updated we trigger an image deploy

Reply:
this makes sense!